samples/pmic/npmx/buck/README.rst

```diff
@@ -0,0 +1,101 @@
+.. _nPM1300 BUCK:
+
+nPM1300 BUCK
+############
+
+Overview
+********
+
+This sample demonstrates the usage of nPM1300 PMIC BUCKs.
+For testing connect voltage multimeter to VOUT1 (output voltage pin of BUCK1). In this example
+voltage can be changed only for BUCK1. BUCK2 voltage is set by default to 3.0V.
+In main.c file select your test case to be executed.
+You can also modify expected output voltages and selected GPIOs to control the system.
+
+Testcases
+---------
```

Comment:
```suggestion
Building and running
**********************
```

Reply:
Based on our [template](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html), please decide what should be called **Testing** indeed, and what should be placed in **Building and running** part.